.env.example

```diff
@@ -4,4 +4,7 @@ DATABASE_PASSWORD=postgres
 DATABASE_PORT=5432
 DATABASE_NAME=postgres
 
-STATIC_WORKER_POOL=5
\ No newline at end of file
+STATIC_WORKER_POOL=5
+WHITE_LIST_PATH=/home/user/go/src/github.com/ufcg-lsd/arrebol-pb/whitelist
```

Comment:
what is this WHITE_LIST? how is this related to the arrebol new design?

Reply:
I've requested @wesleymonte to add this white_list. Btw, we should rename it to avoid discriminatory terms. The meaning of this has been discussed in one of our telcos, and it stands for the list of the workers' ids generated and allowed by the server. This way, only workers whose admins have access to the server can join the server with a valid ID, which avoid some intruders. When we have the ResourceManager, it will be able to acess these ids, and if someone need to deploy a worker by hand, a valid ID is required.